File: B/Python/CPU.py
class Processor:
    __name = ""
    __priceCPU = 0
    
    def __init__(self, name, price):
        self.__name = name
        self.__priceCPU = price
    
    def setName(self, b):
        self.__name = b
    
    def getName(self):
        return self.__name
    
    def setPriceCPU(self, m):
        self.__priceCPU = m
    
    def getPriceCPU(self):
        return self.__priceCPU
    
    def printCPU(self):
        print("CPU           : "+str(self.__name))

File: B/PHP/CPU.php
<?php

class CPU{
    private $name = 0;
    private $priceCPU = 0;

    public function __construct($name = 0, $priceCPU = 0){
        $this->name = $name;
        $this->priceCPU = $priceCPU;
    }

    public function setName($name){
        $this->name = $name;
    }

    public function getName(){
        return $this->name;
    }

    public function setPriceCPU($p){
        $this->priceCPU = $p;
    }

    public function getPriceCPU(){
        return $this->priceCPU;
    }

    public function printCPU(){
        echo "CPU          : ". $this->name . "<br>";
    }
}

?>